tq42/channel.py

```diff
@@ -114,17 +124,46 @@ async def _handle():
                     f"Channel was closed due to exceeding {message_timeout_in_sec}s timeout between messages"
                 ) from None
 
+            except grpc.aio.AioRpcError as e:
+                if e.code() in _RETRYABLE_ERROR_CODES:
+                    call = await self._reestablish_connection()
+                    if call:
+                        return await _handle()
+
+                raise e
+
+        call = await self._establish_connection()
```

Comment:
I know it is definitely not necessary but just for readability, shouldn't this line be above the two inner function definitions?